File: 1101.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, m, max, i;
    while(true)
    {
        cin>>n>>m;
        if(n<=0)break;
        if(m<=0)break;

        if(n>m)
        {
            max=n;
            n=m;
            m=max;
        }
        int sum=0;
        for(i=n; i<=m; i++)
        {
            cout<<i<<" ";
            sum+=i;

        }
        cout<<"Sum="<<sum<<endl;



    }



    return 0;
}

File: 1101.py
while(True):
    a, b = map(int, input().split())
    x = min(a, b)
    y = max(a, b)
    if a<=0 or b<=0:
        break
    else:
        result = []
        for i in range(x, y+1):
            result.append(i)
            final = ' '.join(map(str, result))
    print(f"{final} Sum={sum(result)}")



